src/cli.rs

```diff
@@ -299,6 +299,16 @@ pub enum EdgeAppCommands {
         #[arg(short, long)]
         path: Option<String>,
 
+        /// Edge app id. If not specified CLI will use the id from the manifest.
+        #[arg(short, long)]
+        app_id: Option<String>,
+    },
+    /// Deletes an edge app. This cannot be undone.
+    Delete {
```

Comment:
Why is it not taking id optionally?

Reply:
/// Edge App id. If not specified CLI will use the id from the manifest.
        #[arg(short, long)]
        app_id: Option<String>,

Reply:
It is

Reply:
my bad, it's because how diff shows - did not notice